core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -647,6 +687,34 @@ case object ModelInsights {
     }
   }
 
+  private[op] def descaleLRContrib(
+    model: Option[Model[_]],
+    sparkFtrContrib: Seq[Double],
+    featureStd: Double,
+    labelStd: Double): Option[Seq[Double]] = {
+    val stage = model.flatMap {
+      case m: SparkWrapperParams[_] => m.getSparkMlStage()
+      case _ => None
+    }
+    stage.collect {
+//      case m: LogisticRegressionModel =>
+//        if (m.getStandardization && sparkFtrContrib.nonEmpty) {
+//          // scale entire feature contribution vector
+//          sparkFtrContrib.map(_ * featureStd)
+//        }
+//        else sparkFtrContrib
+      case m: LinearRegressionModel =>
+        if (m.getStandardization && sparkFtrContrib.nonEmpty) {
+          // need to also divide by labelStd for linear regression
+          // See https://github.com/salesforce/TransmogrifAI/pull/345#discussion_r303634114
+          // Also see https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala#L551-L558
```

Comment:
I meant a paper citation. How did you arrive at this formula? @Jauntbox any comments?

Reply:
I reverse engineered Spark's implementation (link to their implementation is included in the code comment).

Reply:
Will add a formal paper citation if i can find one

Reply:
In formal literature, this is referred to as standardized (regression) coefficients, or beta coefficients. See https://en.wikipedia.org/wiki/Standardized_coefficient. The two citations in the Wiki page are not open access, would you still want me to include those? Informal citations are https://www3.nd.edu/~rwilliam/stats1/x92.pdf or https://u.demog.berkeley.edu/~andrew/teaching/standard_coeff.pdf

For standardizing coeffs in logistic regression, see this blog: https://think-lab.github.io/d/205/. Tldr, they suggest using the Agresti approach (what I implemented, but currently commented out) -- which is multiplying the coeff by the respective standard deviation of the feature.

Reply:
Yeah, I just double-checked the math. If you do the zero mean and unit std dev rescaling, then both the features and label should scale as x' = (x - mean(x))/std(x).

If you plug the scaled values into the linear regression equation, then the weights in the unscaled space are weight_unscaled(i) = std(label) * weight_scaled(i) / std(i)

So the weights in the scaled space are weight_scaled(i) = weight_unscaled(i) * std(i) / std(label), which is what @TuanNguyen27 has. Is there a specific example you're worried this is incorrect for @leahmcguire?

Reply:
Ok please add those citations @TuanNguyen27. And thanks for double checking @TuanNguyen27 and @Jauntbox!